InterventionSkillExample/skill.yaml

```diff
@@ -0,0 +1,46 @@
+camel: 1.0.0
+name: intervention-example
+title: Intervention Example
+description: Intervention Skill Example
```

Comment:
Add a more detailed description than this.  Intervention Skill is used by Mission Planner ..... It takes in ABC parameter and returns EFG parameter.